src/components/app.jsx

```diff
@@ -48,10 +48,9 @@ class App extends React.PureComponent {
           <h2 className="ds-h2">Explore the API using the public sandbox.</h2>
           <p className="ds-text--lead">The public sandbox API allows you to test integrating with the API before the reporting period begins - without making a real submission to QPP. You can access the public sandbox API endpoints via the command line using the URL 'https://qpp-submissions-sandbox.navapbc.com/v1/'. Check out the <a href="https://qpp-submissions-sandbox.navapbc.com/">interactive API reference</a> for an exhaustive list of endpoints with example request and response payloads.</p>
 
-          <h2 className="ds-h2">Participate in the Submissions API Private Beta</h2>
-          <p className="ds-text--lead">The QPP Submissions API will be in private beta from July 2017 through December 2017. During this period, we will collect participants' feedback and make refinements before we open the API for public use.</p>
-          <p className="ds-text--lead">At this time, the private beta is open only to 2017 CMS-approved Qualified Registries ("registries") and Qualified Clinical Data Registries ("QCDRs").  To participate, you will need to request an API key by contacting the Quality Payment Program Service Center at <a href="mailto:qpp@cms.hhs.gov?subject=Request%20API%20Key%20for%20Submissions%20API%20Beta">QPP@cms.hhs.gov</a>.</p>
-          <a className="ds-c-button ds-c-button--primary" href="/qpp-submissions-docs/privatebeta">Learn more about the private beta</a>
+          <h2 className="ds-h2">Participate in the Submissions API Developer Preview</h2>
+          <p className="ds-text--lead">CMS is inviting developers to provide feedback on the new Submissions API for the purpose of improved development. From July 2017 through December 2017, participants in the Developer Preview can build direct integrations with other software to test programmatically submitting Quality Payment Program (QPP) performance data to CMS.</p>
+          <a className="ds-c-button ds-c-button--primary" href="/qpp-submissions-docs/privatebeta">Learn more about the Developer Preview</a>
```

Comment:
we should update the link here and the component name to 'developer-preview' instead of 'privatebeta'. If there have been email blasts including the privatebeta link, we can also route 'privatebeta' in app.jsx to the 'developer-preview' component.